### PR TITLE
Adjust CI for auto-installing SMCP and SMMR

### PR DESCRIPTION
**Pull request by navidshaikh**
- Removes manual configuration of ServiceMeshMembeRoll and ServiceMeshControlPlane
 - SMMR and SMCP should be handled automatically by serverless-operator
 - All the kn e2e tests namespaces will be configured as members of SMMR

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -125,49 +86,6 @@ function timeout() {
...
-  oc apply -n $SERVICEMESH_NAMESPACE -f https://raw.githubusercontent.com/openshift/knative-serving/$SERVING_RELEASE/openshift/servicemesh/controlplane-install.yaml
-  cat <<EOF | oc apply -f -
-apiVersion: maistra.io/v1
-kind: ServiceMeshMemberRoll
-metadata:
-  name: default
-  namespace: ${SERVICEMESH_NAMESPACE}
-spec:
-  members:
-  - ${SERVING_NAMESPACE}
```
Comment (mgencur) [inline openshift/e2e-tests-openshift.sh]: How are you adding the test namespace now? Are they added automatically?

Comment (navidshaikh) [reply openshift/e2e-tests-openshift.sh]: Yes, they are being added automatically.

Comment (mgencur) [reply openshift/e2e-tests-openshift.sh]: Interesting!

**Pull request merged**